doc/faq/howto_faq.rst

```diff
@@ -657,14 +656,13 @@ Agg is to call::
     matplotlib.use('Agg')
     import matplotlib.pyplot as plt
 
-For more on configuring your backend, see
-:ref:`what-is-a-backend`.
+For more on configuring your backend, see :ref:`what-is-a-backend`.
 
 Alternatively, you can avoid pylab/pyplot altogether, which will give
 you a little more control, by calling the API directly as shown in
 :doc:`/gallery/user_interfaces/canvasagg`.
 
-You can either generate hardcopy on the filesystem by calling savefig::
+You can either generate hardcopy on the filesystem by calling `savefig`::
```

Comment:
This does not link because it's ambiguous (Figure/pyplot). Use `.Figure.savefig` or `~.Figure.savefig`.

Reply:
fixed